ansible/configs/ocp-storage-cns/post_software.yml

```diff
@@ -81,14 +81,17 @@
         - gen_user_vol_pv
         - openshift_nfs_config
 
-    - shell: 'oc create -f /root/pvs-{{ env_type }}-{{ guid }}.yml || oc replace -f /root/pvs-{{ env_type }}-{{ guid }}.yml'
-      tags:
-        - create_user_pv
+    - shell: >-
+        oc create -f /root/pvs-{{ env_type }}-{{ guid }}.yml
+        || oc replace -f /root/pvs-{{ env_type }}-{{ guid }}.yml
       when: pv_list.0 is defined
       tags:
+        - create_user_pv
```

Comment:
This tag was not in the original, but it was added here?

Reply:
good catch